source/installguide/hypervisor/kvm.rst

```diff
@@ -613,7 +613,7 @@ Modify the interfaces file to look like this:
    auto cloudbr0
    iface cloudbr0 inet static
        bridge_ports eth0
-       bridge_fd 5
+       bridge_fd 0
```

Comment:
This is `how long` it'll be before the interface will be able to do anything. During this time the bridge will be discovering other bridges and checking that no loops are created. For a better description and the reason for it you need to read up on spanning tree protocol.
ref: https://serverfault.com/questions/39298/what-is-bridge-fd

Q - does `0` mean, infinite or just 0 (don't wait?)

Reply:
I think this is `dont wait`, generally an inifinte value is represented by a negative value